src/pages/Script/ScriptsComp.tsx

```diff
@@ -39,14 +39,14 @@ export const ScriptTransactions = ({ page, size }: { page: number; size: number
         },
       })
       .then((res: AxiosResponse) =>
-        toCamelcase<Response.Response<{ ckbTransactions: CkbTransactionInScript[] }>>(res.data),
+        toCamelcase<Response.Response<{ ckbTransactions: CkbTransactionInScript[]; meta: PageInfo }>>(res.data),
```

Comment:
Strange, why is the backend API using this response format?

Based on the type, shouldn't `meta` have been provided at the root level before?

Reply:
cc @rabbitz

Reply:
> Strange, why is the backend API using this response format?
> 
> Based on the type, shouldn't `meta` have been provided at the root level before?

`data` maybe the root level of the response structure, with other properties like `message` or `status` always for restful API.

Reply:
https://github.com/nervosnetwork/ckb-explorer/commit/04b8ae9ff618254fd8c566afb9b21f19d1fd1094#diff-09a97666b995d34f0472095b2c99024773a75e4640cba18e3510085e5ed315f2R27

This return data structure was defined by the previous author and has been deployed for the interface. I assumed it was an agreed-upon structure with the frontend.

Reply:
At the time, we discussed the response structure convention during a meeting and in https://github.com/Magickbase/ckb-explorer-public-issues/issues/315#issuecomment-1541319430. At that time, it was agreed that meta would be directly under the root level.

Reply:
> At the time, we discussed the response structure convention during a meeting and in [Magickbase/ckb-explorer-public-issues#315 (comment)](https://github.com/Magickbase/ckb-explorer-public-issues/issues/315#issuecomment-1541319430). At that time, it was agreed that meta would be directly under the root level.

If so, the API should be adjust.

Reply:
For this issue, another issue needs to be created to be handled by the backend.
https://github.com/Magickbase/ckb-explorer-public-issues/issues/451